### PR TITLE
Feature/915/workflow search filters

### PR DESCRIPTION
**Pull request by garyluu**
This is supposed to be merged in develop, but base is a different one to show differences.

This adds a few search facets to workflows (verified, verified source, organization).  It also denotes which facet is entry type specific.  It currently does not combine tool facets and workflow facets together even if they are similar.

### CODE REVIEW

File: cypress/plugins/index.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+// ***********************************************************
```
Comment (denis-yuen) [inline cypress/plugins/index.js]: Is a cypress change intentional?

File: src/app/container/paramfiles/paramfiles.service.ts [hunk 2]
```diff
@@ -32,7 +32,7 @@ export class ParamfilesService {
     if (type === 'workflows') {
       return this.workflowsService.getTestParameterFiles(id, versionName);
     } else {
-      return this.containersService.getTestParameterFiles(id, versionName, descriptor);
+      return this.containersService.getTestParameterFiles(id, descriptor, versionName);
```
Comment (denis-yuen) [inline src/app/container/paramfiles/paramfiles.service.ts]: Why is this reversing in this PR?

File: src/app/search/search.service.ts [hunk 3]
```diff
@@ -201,17 +201,20 @@ export class SearchService {
...
       ['Language', 'descriptorType'],
       ['Registry', 'registry'],
       ['Private Access', 'private_access'],
       ['Verified', 'tags.verified'],
       ['Author', 'author'],
-      ['Organization', 'namespace'],
+      ['Namespace', 'namespace'],
       ['Labels', 'labels.value.keyword'],
       ['Verified Source', 'tags.verifiedSource'],
+      ['Verified 2', 'workflowVersions.verified'],
```
Comment (denis-yuen) [inline src/app/search/search.service.ts]: These names seem pretty non-descriptive (namespace, organization, verified 2, verified source 2). What is this a map used for (i.e. this map could use a code comment as well)

File: cypress/plugins/index.js (hunk 1 above)
Comment (garyluu) [reply cypress/plugins/index.js]: New cypress allows plugins, so checked in the empty plugins file in the event that we do use plugins in the future.  Prevents unchecked file locally.

File: src/app/container/paramfiles/paramfiles.service.ts (hunk 2 above)
Comment (garyluu) [reply src/app/container/paramfiles/paramfiles.service.ts]: Looks like swagger codegen or the swagger.yaml reversed the parameters.

Comment (denis-yuen) [reply src/app/container/paramfiles/paramfiles.service.ts]: It doesn't seem like either or those are changing in this PR though, so was this previous broken and undetected or was there a non-checked-in change here?

Comment (garyluu) [reply src/app/container/paramfiles/paramfiles.service.ts]: Previous broken and undetected

Comment (denis-yuen) [reply src/app/container/paramfiles/paramfiles.service.ts]: Ok, might be time to think of a test or extend the parameter ordering that was put in earlier.
This seems like an easy way to introduce an error without noticing

**Pull request closed without merging**